packages/test/README.md

```diff
@@ -0,0 +1,196 @@
+# @theforeman/test
+
+> Adds theforeman testing tools to you project.
+
+[![Package Version](https://img.shields.io/npm/v/@theforeman/test.svg?style=flat-square)](https://www.npmjs.com/package/@theforeman/test)
+[![Downloads Status](https://img.shields.io/npm/dm/@theforeman/test.svg?style=flat-square)](https://npm-stat.com/charts.html?package=@theforeman/test&from=2016-04-01)
+[![Build Status: Linux](https://img.shields.io/travis/theforeman/foreman-js/master.svg?style=flat-square)](https://travis-ci.org/theforeman/foreman-js)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+
+## Installation
+
+```sh
+npm install --save-dev @theforeman/test
+```
+
+1. Use `tfm-test` and `tfm-coveralls` (for coverage) scripts under `test` and `coveralls` respectively in `package.json`:
+
+```json
+{
+  "test": "tfm-test --plugin",
+  "coveralls": "tfm-coveralls"
```

Comment:
`coveralls` is the name of the specific tool (like jest/eslint).
Please change to `tfm-publish-coverage`

Reply:
You forget to update the readme